src/sentry/tasks/store.py

```diff
@@ -35,7 +35,7 @@ def preprocess_event(cache_key=None, data=None, start_time=None, **kwargs):
 
     if data is None:
         metrics.incr('events.failed', tags={'reason': 'cache', 'stage': 'pre'})
-        logger.error('Data not available in preprocess_event (cache_key=%s)', cache_key)
+        logger.error('preprocess.failed.empty', extra={'redis_key': cache_key})
```

Comment:
Yeah, our implementation that we run for sentry.io is redis, but someone else could run something else.